### PR TITLE
Portfolio-project

### PR DESCRIPTION
**Pull request by micheaol**
In this pull request I add function:
1. When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
2. When the user clicks (or taps) the close (X) button, the popup disappears.
3. Create a pop-up for both Desktop and mobile versions.
4. I store the pieces of data in a JavaScript object.
5. I update the main page so that the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

File: script/main.js [hunk 1]
```diff
@@ -54,3 +123,139 @@ menuContact.addEventListener('click', () => {
...
+      const innerPopUpDiv = createMyElement('div');
+      innerPopUpDiv.className = 'inner-pop-up-div';
+
+      const techLangInnerDiv = createMyElement('div');
+      techLangInnerDiv.className = 'tech-lang-inner-div';
+
+      const popUpTitle = createMyElement('h1');
+      popUpTitle.className = 'pop-up-title';
+      popUpTitle.textContent = project.title;
+      techLangInnerDiv.appendChild(popUpTitle);
```
[the remark this answers is not in the document]
Comment (micheaol) [reply script/main.js:188]: Please find below the image of the project on the screen the technologies are well aligned as the original.

alt [
![popUp
]([]([](text](https://github.com/micheaol/Portfolio-project/blob/dynamic-pop-up-window/images/popUp.PNG))))

File: script/main.js [hunk 2]
```diff
@@ -1,7 +1,76 @@
...
+    tips: "A daily selection of privately personalized reads; no accounts or      sign-ups required. has been the industry's standard",
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    projectImg: '../images/pop-up-img.png',
+    liveVersion: 'https://micheaol-portfolio.netlify.app/',
+    gitRepo: 'https://github.com/micheaol/Portfolio-project',
+    techologies: ['HTML', 'Bootstrap', 'Roby on Rails'],
+  },
+];
+
```
Comment (od-c0d3r) [inline script/main.js:73]: Good Job fulfill the requirement of the previous reviewer 💯 ✅

File: script/main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (od-c0d3r) [reply script/main.js:188]: @micheaol The issue has been clarified in the review. Please make sure to deploy your last changes to your host. 💾

**Review by od-c0d3r — CHANGES_REQUESTED** [review]
Hi @micheaol, @Lino09 and @nicupop729 👋🏽😄 I'm Omar, your CR 

When I first saw your conversion with the previous CR, I inspected your GitHub deployment activity and I found that you didn't deploy your recent changes to the repo since yesterday, and I believe that you made changes hours ago as I see from your commits info. 

e7ab1fd6-7010-45bc-853f-1fd082cf3c83

However, great work on making the changes requested by the previous reviewer 👏 , I hosted your app and reviewed your code, it's working as expected 😎👍🏽 . there is nothing that can stop you now from approval only needs to fix the deployment issue.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- _**[OPTIONAL]**_ It would be great if you use the README template provided by micorverse and you can access it from [here](https://github.com/microverseinc/readme-template)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**